_posts/2022-03-22-sdílejte-svou-praxi-v-otevřených-datech.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Sdílejte svou praxi v otevřených datech!
+detail: true
+ref: PraxeOD
```

Comment:
To neni moc dobrý jednoznačný identifikátor - chtělo by to trochu specifikovat. Co když budeme chtít PraxeOD použít ještě jinde? :)